CMakeLists.txt

```diff
@@ -103,7 +103,6 @@ else()
   set(ONNX_NAMESPACE "onnx")
   set(MY_ONNX_NAMESPACE "-DONNX_NAMESPACE=onnx")
 endif()
-add_definitions(${MY_ONNX_NAMESPACE})
 
 if(ONNX_ML)
   add_definitions("-DONNX_ML=1")
```

Comment:
There is also an `add_definitions` here, but I don't know whether it is ok to replace it.